config.js

```diff
@@ -515,6 +515,17 @@ var config = {
     //    '__end'
     // ],
 
+    // Whether toolbar should be always visible or should hide after x miliseconds.
+    // toolbarAlwaysVisible: false,
+
+    // Moved from interfaceConfig.INITIAL_TOOLBAR_TIMEOUT
+    // The initial numer of miliseconds for the toolbar buttons to be visible on screen.
+    toolbarInitialTimeout: 20000,
+
+    // Moved from interfaceConfig.TOOLBAR_TIMEOUT
+    // Number of miliseconds for the toolbar buttons to be visible on screen.
+    toolbarTimeout: 4000,
```

Comment:
If these values are the default one, we may want to comment these and leave them as a reference for those wanting to change them.

Reply:
Yes please, no uncommented values in config.js

Reply:
done